hadoop-ozone/s3gateway/src/main/java/org/apache/hadoop/ozone/s3/util/OzoneS3Util.java

```diff
@@ -38,9 +38,9 @@
   private OzoneS3Util() {
   }
 
-  public static String getVolumeName(String userName) {
+  public static String getS3Username(String userName) {
```

Comment:
Can we rename this getVolumeName, as it is converting to md5Hex, and we use that as volume name.

Reply:
I am not sure what is the right name, I renamed after a long debug session when I was totally confused what is in a variable.

This method doesn't return with a volumeName it returns with and unique (and safe) identifier of the s3 user. It will be converted to an s3 volume name on the server side ("s3" + this unique id).

This string is used as a parameter of the `createS3Bucket(String userName, String bucketName)` which clearly shows that this is not a volumeName. This is a unique ID / username.

During the debug I was very confused as I expected to have the final volumeName from this method, but didn't get it.

But please let me know if you have any better suggestion for the name.

Reply:
Agreed. I am fine with current way.